support/standalone/standalone.c

```diff
@@ -61,49 +62,53 @@ void crypto_standalone_to_lower(char* str)
 void crypto_standalone_print_help(void)
 {
     printf(CRYPTO_PROMPT "command [args]\n"
-            "----------------------------------------------------------------------\n"
-            "exit                               - Exit app                         \n"
-            "help                               - Display help                     \n"
-            "noop                               - No operation command to device   \n"
-            "reset                              - Reset CryptoLib                  \n"
-            "tc                                 - Toggle TC debug prints           \n"
-            "vcid #                             - Change active TC virtual channel \n"
-            "\n"
-        );   
+                         "----------------------------------------------------------------------\n"
+                         "exit                               - Exit app                         \n"
+                         "help                               - Display help                     \n"
+                         "noop                               - No operation command to device   \n"
+                         "reset                              - Reset CryptoLib                  \n"
+                         "tc                                 - Toggle TC debug prints           \n"
+                         "tm                                 - Toggle TM debug prints           \n"
+                         "vcid #                             - Change active TC virtual channel \n"
```

Comment:
This line spawned a thought, how would we change the active TM virtual channel in FSW?